src/plugins/data/public/ui/dataset_selector/configurator.tsx

```diff
@@ -80,6 +83,11 @@ export const Configurator = ({
       setTimeFields(dateFields || []);
     };
 
+    if (baseDataset?.dataSource?.meta?.supportsTimeFilter === false) {
+      setTimeFields([]);
```

Comment:
this `[]` creates a different object every time when called, and will trigger a re-render. might be better to add a guard
```suggestion
      if (timeFields.length > 0) setTimeFields([]);
```

Reply:
Updated here: [53198c4](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8609/commits/53198c40ecdcea449979f3eb51eee7782300cb54)